Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-core/kodi-main/) for Kodi core/kodi main.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-core/-/kodi-main/horizontal-auto.svg)

gade01 commented:
I'm doing a lot of language fixes at Weblate.

Fixes:
- https://github.com/xbmc/xbmc/issues/18767
- https://github.com/xbmc/xbmc/issues/19536
- https://github.com/xbmc/xbmc/issues/18712
- https://github.com/xbmc/xbmc/issues/19389
- https://forum.kodi.tv/showthread.php?tid=362141
- https://forum.kodi.tv/showthread.php?tid=361007

weblate committed:
Translated using Weblate (German (de_de))

Currently translated at 100.0% (4143 of 4143 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 95.3% (3951 of 4143 strings)

Translated using Weblate (Esperanto)

Currently translated at 23.9% (991 of 4143 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (4143 of 4143 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 47.6% (1976 of 4143 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 99.0% (4105 of 4143 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 99.0% (4105 of 4143 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 71.4% (2961 of 4143 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (4143 of 4143 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (4143 of 4143 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 82.6% (3425 of 4143 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 90.2% (3739 of 4143 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 99.5% (4124 of 4143 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 82.6% (3423 of 4143 strings)

Translated using Weblate (Telugu (India) (te_in))

Currently translated at 9.5% (396 of 4143 strings)

Translated using Weblate (Tamil (India) (ta_in))

Currently translated at 54.9% (2276 of 4143 strings)

Translated using Weblate (Silesian)

Currently translated at 82.1% (3402 of 4143 strings)

Translated using Weblate (Serbian (latin))

Currently translated at 82.8% (3432 of 4143 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 92.6% (3838 of 4143 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 49.5% (2051 of 4143 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 71.8% (2976 of 4143 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 96.4% (3997 of 4143 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 30.0% (1245 of 4143 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 98.9% (4100 of 4143 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 97.7% (4051 of 4143 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 95.6% (3961 of 4143 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 95.6% (3964 of 4143 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 98.8% (4097 of 4143 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 92.4% (3831 of 4143 strings)

Translated using Weblate (Burmese (my_mm))

Currently translated at 30.8% (1280 of 4143 strings)

Translated using Weblate (Maltese (mt_mt))

Currently translated at 70.6% (2929 of 4143 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 95.1% (3943 of 4143 strings)

Translated using Weblate (Mongolian (mn_mn))

Currently translated at 19.2% (796 of 4143 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 63.7% (2642 of 4143 strings)

Translated using Weblate (Maori)

Currently translated at 53.0% (2199 of 4143 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 86.7% (3595 of 4143 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 98.6% (4089 of 4143 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (Icelandic (is_is))

Currently translated at 96.6% (4006 of 4143 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 71.3% (2957 of 4143 strings)

Translated using Weblate (Armenian (hy_am))

Currently translated at 14.7% (612 of 4143 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 99.6% (4127 of 4143 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 95.1% (3942 of 4143 strings)

Translated using Weblate (Hindi (India) (hi_in))

Currently translated at 18.9% (784 of 4143 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 93.8% (3889 of 4143 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 95.6% (3963 of 4143 strings)

Translated using Weblate (Faroese (fo_fo))

Currently translated at 40.9% (1698 of 4143 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 66.1% (2741 of 4143 strings)

Translated using Weblate (Persian (Afghanistan) (fa_af))

Currently translated at 6.6% (276 of 4143 strings)

Translated using Weblate (Esperanto)

Currently translated at 24.1% (999 of 4143 strings)

Translated using Weblate (German (de_de))

Currently translated at 99.7% (4132 of 4143 strings)

Translated using Weblate (Welsh (United Kingdom) (cy_gb))

Currently translated at 69.8% (2894 of 4143 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 83.3% (3454 of 4143 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 44.9% (1863 of 4143 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 99.7% (4131 of 4143 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 96.9% (4016 of 4143 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 97.8% (4055 of 4143 strings)

Update translation files

Updated by "Source update" hook in Weblate.

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 99.9% (4021 of 4022 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 99.8% (1976 of 1979 strings)

Translated using Weblate (Chinese (Taiwan) (zh_tw))

Currently translated at 100.0% (4043 of 4043 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 100.0% (4131 of 4131 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (3964 of 3964 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (3964 of 3964 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Chinese (Taiwan) (zh_tw))

Currently translated at 100.0% (4043 of 4043 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 100.0% (3425 of 3425 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 100.0% (3739 of 3739 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 99.8% (4129 of 4137 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 100.0% (3423 of 3423 strings)

Translated using Weblate (Telugu (India) (te_in))

Currently translated at 100.0% (396 of 396 strings)

Translated using Weblate (Tamil (India) (ta_in))

Currently translated at 100.0% (2276 of 2276 strings)

Translated using Weblate (Silesian)

Currently translated at 100.0% (3402 of 3402 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (4110 of 4110 strings)

Translated using Weblate (Serbian (latin))

Currently translated at 100.0% (3432 of 3432 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 100.0% (3838 of 3838 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 100.0% (2051 of 2051 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 100.0% (2976 of 2976 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 100.0% (3997 of 3997 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 100.0% (1245 of 1245 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (4003 of 4003 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (4003 of 4003 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 100.0% (4051 of 4051 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 100.0% (3961 of 3961 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (3964 of 3964 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 100.0% (4102 of 4102 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 100.0% (3831 of 3831 strings)

Translated using Weblate (Burmese (my_mm))

Currently translated at 100.0% (1280 of 1280 strings)

Translated using Weblate (Mongolian (mn_mn))

Currently translated at 100.0% (796 of 796 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 100.0% (2642 of 2642 strings)

Translated using Weblate (Maori)

Currently translated at 100.0% (2199 of 2199 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 100.0% (3595 of 3595 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 100.0% (4093 of 4093 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 99.8% (2957 of 2961 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 100.0% (4132 of 4132 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 100.0% (3942 of 3942 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 100.0% (3889 of 3889 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 100.0% (3963 of 3963 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 100.0% (3963 of 3963 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 100.0% (2741 of 2741 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 100.0% (3674 of 3674 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (4120 of 4120 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 100.0% (3388 of 3388 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 100.0% (3326 of 3326 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (4137 of 4137 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 100.0% (3930 of 3930 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (4129 of 4129 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 100.0% (4136 of 4136 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 100.0% (4022 of 4022 strings)

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 100.0% (2208 of 2208 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 100.0% (1979 of 1979 strings)

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Jeremy Chen <jeremy@naviart.idv.tw>
Co-authored-by: Kai Sommerfeld <kai.sommerfeld@gmx.com>
Co-authored-by: Meano Lee <meanocat@gmail.com>
Co-authored-by: Raimondas Dužinskas <raimondas.duzinskas@gmail.com>
Co-authored-by: liimee <alt3753.7@gmail.com>
Co-authored-by: vdkbsd <valexgus@gmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/af_za/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ar_sa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/bg_bg/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/bs_ba/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/cy_gb/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/el_gr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/en_us/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/eo/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/es_ar/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/eu_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/fa_af/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/fa_ir/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/fo_fo/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/fr_ca/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/gl_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/he_il/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/hi_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/hu_hu/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/hy_am/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/is_is/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ja_jp/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/lt_lt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/lv_lv/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/mi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/mk_mk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/mn_mn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ms_my/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/mt_mt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/my_mm/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/nb_no/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/pt_pt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/si_lk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/sk_sk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/sl_si/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/sq_al/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/sr_Latn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/sr_rs/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/szl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/ta_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/te_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/th_th/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/vi_vn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/zh_tw/
Translation: Kodi core/kodi main
